qpc/cred/tests_cred_edit.py

```diff
@@ -180,14 +179,14 @@ def test_edit_vcenter_cred(self):
         url_patch = get_server_location() + CREDENTIAL_URI + '1/'
         results = [{'id': 1, 'name': 'cred1',
                     'cred_type': VCENTER_CRED_TYPE, 'username': 'root',
-                    'password': '********'}]
+                    'sshkeyfile': '/foo/bar'}]
         data = {'count': 1, 'results': results}
         with requests_mock.Mocker() as mocker:
             mocker.get(url_get, status_code=200, json=data)
             mocker.patch(url_patch, status_code=200)
             aec = CredEditCommand(SUBPARSER)
             args = Namespace(name='cred1', username='root',
-                             password=None)
+                             sshkeyfile='/foo/bar')
```

Comment:
My mistake 🤦‍♀️. I was trying to make these make more sense because they are essentially the same test as the one below them except unsetting the password so I was trying to instead add an ssh-keyfile but I overlooked that it was vcenter.